src/test/java/giis/eshopcontainers/e2e/functional/model/EShopUser.java

```diff
@@ -0,0 +1,33 @@
+package giis.eshopcontainers.e2e.functional.model;
```

Comment:
As we are doing tests, not replicating a domain model. Is there any reason to add a class like this?